flox-bash/lib/templateFloxEnv/pkgs/default/flox.nix

```diff
@@ -4,16 +4,31 @@
   # packages.nixpkgs-flox.figlet = {};
   # packages.nixpkgs-flox.bat = { version = "0.22.1"; };
 
+  # Activation Extensions
+  # Provides an extensible `<env>/etc/profile` script.
+  # packages.flox.etc-profiles = {
+  #   # All language packs are installed by default, but you can also
+  #   # select individual packs by uncommenting the line below.
+  #   # Invoke `flox search -c flox etc-profiles -l` to see
+  #   # a list of all supported language pack outputs.
+  #   # Please note that all/most language packs depend on including
+  #   # the "base" and "common_paths" output.
```

Comment:
Is there an error if they are not present, how can I tell if I didn't include them?

Reply:
As of right now it would just come from the errors printed to stderr at startup. On one of my previous iterations I embedded the contents of both of these into the `base` output which was then linked into every other package, but then I was reminded of Alex's requirement that users be able to easily swap out any part of it. FWIW I expect most people will just install the default `out` output which includes all language packs.